packages/ui-react/src/components/LanguageMenu/LanguageMenu.module.scss

```diff
@@ -30,3 +31,9 @@
     font-weight: bold;
   }
 }
+
+.iconButton {
+  width: 36px;
+  height: 36px;
+  margin-left: calc(#{variables.$base-spacing} / 2);
+}
```

Comment:
I think this is good enough for now. The root cause is that the iconbutton styling is moved from `Header.module.scss` to `LanguageMenu.module.scss`. There are of course multiple ways to approach this problem, but I think it's good to leave it at this pragmatic solution for now.

Reply:
Agreed, I aimed for minimal code to replicate the look. I've also created a ticket for the highlighting as discussed. - [OTT-538](https://videodock.atlassian.net/browse/OTT-538)